app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,26 @@
+<h2>Log in</h2>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <div class="field">
+    <%= f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email" %>
+  </div>
+
+  <div class="field">
+    <%= f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password" %>
+  </div>
+
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+
+  <div class="actions">
+    <%= f.submit "Log in" %>
+  </div>
+<% end %>
+
+<%= render "devise/shared/links" %>
```

Comment:
- [ ] [REQUIRED] Great job with the devise implementation, however, it is a hard requirement to ensure that your design for the REGISTRATION(Signup) and LOGIN page be modified to match the wireframes provided and their styling. Let's fix this so it looks like the wireframe.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="744" alt="Screenshot 2023-07-27 at 21 19 23" src="https://github.com/smunoz1988/blog-on-rails/assets/58692590/39d1a3c8-d0bc-4782-abbe-121bf74cd536"> | <img width="1031" alt="Screenshot 2023-07-27 at 13 26 52" src="https://github.com/smunoz1988/blog-on-rails/assets/58692590/3a2d16ab-801e-4ce0-8ece-4f04ea196f69">

Reply:
@Whoistolu  Done! thanks for the revision!